### PR TITLE
Feature/dp add deploy setup scripts

### PR DESCRIPTION
**Pull request by DenisPopov15**

### CODE REVIEW

File: bin/deploy_heroku.bash [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```
Comment (alexkravets) [inline bin/deploy_heroku.bash]: Please rename to `deploy-heroku.sh` and use `#!/bin/sh` — this is more common.

Comment (alexkravets) [inline bin/deploy_heroku.bash]: Actually rename to just `deploy-heroku`.

File: bin/deploy_heroku.bash [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+echo "Deploy to heroku"
+echo "Enter name of app "
+read APP_NAME
```
Comment (alexkravets) [inline bin/deploy_heroku.bash]: APP_NAME should be passed as parameter while running script.

File: bin/deploy_heroku.bash [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+SENDGRID_USERNAME=$(heroku config:get SENDGRID_USERNAME)
+SENDGRID_PASSWORD=$(heroku config:get SENDGRID_PASSWORD)
+MONGOLAB_URI=$(heroku config:get MONGOLAB_URI)
+
+heroku config:set SMTP_USERNAME="${SENDGRID_USERNAME}" SMTP_PASSWORD="${SENDGRID_PASSWORD}" MONGODB_URI="${MONGOLAB_URI}"
+
+git push heroku master
+
+echo "AdminUser.create!(name: 'Admin', email: 'user@example.com', password: 'password'); exit" | heroku run rails console
```
Comment (alexkravets) [inline bin/deploy_heroku.bash]: There is a rake task: `rake db:seed` that suppose to do this on production, so you just need to run this rake task on heroku.

File: lib/tasks/setup_s3.rake [hunk 4]
```diff
@@ -0,0 +1,132 @@
+namespace :heroku do
```
Comment (alexkravets) [inline lib/tasks/setup_s3.rake]: Please use `deploy` as namespace, also move this script to `lib/tasks/deploy` folder. Rename script to `create-s3-bucket` rake tasks.

Create sh script `bin/setup-s3`, this should take AWS parameters and pass them to rake task. After bucket is created, use `heroku` to set variables on heroku.

Comment (alexkravets) [inline lib/tasks/setup_s3.rake]: Scrips should not ask user for any parameters, scripts should require them to be passed as parameters for the script.

File: bin/deploy_heroku.bash (hunk 2 above)
Comment (DenisPopov15) [inline bin/deploy_heroku.bash]: @alexkravets Yes, and this script do this, or you mean like ./script -app_name?

File: bin/deploy-heroku [hunk 5]
```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+if [ "$*" == "" ]; then
+    echo "No arguments provided"
```
Comment (alexkravets) [inline bin/deploy-heroku]: "Please provide heroku application name."

**Comment by DenisPopov15** [comment]
@alexkravets updated.

**Pull request closed without merging**